templates/zerver/help/message-a-stream-by-email.md

```diff
@@ -6,7 +6,7 @@ You can send a message to a stream by email by following the following steps.
 {!filter-streams.md!}
 
 2. Select the stream that you want to message by email in the
-[Streams](/#streams) page; {!stream-settings.md!}
+[Streams](https://chat.zulip.org/#streams) page; {!stream-settings.md!}
```

Comment:
This change is not correct, since chat.zulip.org is a single Zulip server, and the software runs on lots of servers.  The relative URL should be correct for reaching that URL on the same server.  Why did you want this change?

Reply:
This doc is hosted on zulipchat.com. When someone clicks on this link, it redirects them to zulipchat.com/#streams. This might cause confusion to some users(already has).
Maybe we could write something like <your-organisation-url>/#streams

Reply:
Ahh, I see, I think basically what we want is for:
* On a root domain that doesn't host an organization, like zulipchat.com itself, no link.
* On a subdomain that does host an organization, like foo.zulipchat.com, the current behavior.

We have some logic for doing that sort of special case for other things in `add_api_uri_context`; probably can use one of the variables defined there.